src/Network/Network/LoadBalancer/NewAzureLoadBalancerBackendPool.cs

```diff
@@ -0,0 +1,167 @@
+using System.Linq;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "LoadBalancerBackendAddressPool"), OutputType(typeof(PSBackendAddressPool))]
+    public partial class NewAzureLoadBalancerBackendPool : NetworkBaseCmdlet
+    {
+        private const string CreateByNameParameterSet = "CreateByNameParameterSet";
+        private const string CreateByParentObjectParameterSet = "CreateByParentObjectParameterSet";
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The resource group name of the load balancer.",
+            ParameterSetName = CreateByNameParameterSet)]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the load balancer.",
+            ParameterSetName = CreateByNameParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string LoadBalancerName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The load balancer resource.",
+            ValueFromPipeline = true,
+            ParameterSetName = CreateByParentObjectParameterSet)]
+        public PSLoadBalancer LoadBalancer { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the backend pool.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The backend addresses.")]
+        [ValidateNotNullOrEmpty]
+        public PSLoadBalancerBackendAddress[] LoadBalancerBackendAddress { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Do not ask for confirmation if you want to overwrite a resource")]
+        public SwitchParameter Force { get; set; }
+
+        public override void Execute()
+        {
+
+            base.Execute();
+
+            if (this.IsParameterBound(c => c.LoadBalancer))
+            {
+                this.ResourceGroupName = this.LoadBalancer.ResourceGroupName;
+                this.LoadBalancerName = this.LoadBalancer.Name;
+            }
+
+            BackendAddressPool existingloadBalancerBackendAddressPool = null;
+
+            // Confirm if resource already exists
+            try
+            {
+                existingloadBalancerBackendAddressPool = this.NetworkClient.NetworkManagementClient.LoadBalancerBackendAddressPools.Get(this.ResourceGroupName, this.LoadBalancerName, this.Name);
+            }
+            catch (Rest.Azure.CloudException exception)
+            {
+                if (exception.Response.StatusCode == System.Net.HttpStatusCode.NotFound)
+                {
+                    existingloadBalancerBackendAddressPool = null;
```

Comment:
this is already null, no need to set it. again. Only throw if other than not found is returned